Sources/XMTP/Codecs/RemoteAttachmentCodec.swift

```diff
@@ -74,7 +74,7 @@ public struct RemoteAttachment {
 
 	func ensureSchemeMatches() throws {
 		if !url.hasPrefix(scheme.rawValue) {
-			throw RemoteAttachmentError.invalidScheme("scheme must be https://")
+			throw RemoteAttachmentError.invalidScheme("scheme must be https")
 		}
 	}
```

Comment:
@nakajima I think this is where the error is getting thrown. Is there a reason we call this on every init?